graph-backup-job/overlays/ocp4-stage/kustomization.yaml

```diff
@@ -23,26 +22,11 @@ patches:
       annotationSelector: "operation=chat-notification"
   - patch: |-
       - op: add
-        path: "/spec/jobTemplate/spec/template/spec/containers/0/env/-1"
-        value: {"name": "THOTH_DATA_DUMP_PATH", "value": "/var/lib/pgsql/pg_dump.sql"}
-    target:
-      group: batch
-      version: v1beta1
-      kind: CronJob
-      name: graph-backup
-  - patch: |-
-      - op: add
-        path: "/spec/jobTemplate/spec/template/spec/containers/0/volumeMounts/-1"
-        value: {"name": "postgresql-data","mountPath": "/var/lib/pgsql"}
-    target:
-      group: batch
-      version: v1beta1
-      kind: CronJob
-      name: graph-backup
-  - patch: |-
+        path: "/spec/jobTemplate/spec/template/spec/containers/0/resources/requests/ephemeral-storage"
+        value: "20Gi"
       - op: add
```

Comment:
```suggestion
      - op: replace
```

Reply:
Both works,  anyhow i have changed it :+1: